driver/src/test/java/org/kaazing/k3po/driver/internal/RobotIT.java

```diff
@@ -207,7 +205,6 @@ public void shouldAbortConnectOK() throws Exception {
         Pattern p = Pattern.compile(expected);
         assertTrue(p.matcher(observedScript).matches());
```

Comment:
Consider using this approach instead to improve readability.
http://stackoverflow.com/questions/8505153/assert-regex-matches-in-junit